spec/github_helper_spec.rb

```diff
@@ -15,83 +34,28 @@
     end
 
     before do
-      allow(described_class).to receive(:github_client).and_return(client)
+      allow(described_class).to receive(:client).and_return(client)
     end
 
     it 'fails if it does not find the right release on GitHub' do
       stub = stub_request(:get, content_url).to_return(status: [404, 'Not Found'])
-      expect(described_class.download_file_from_tag(repository: test_repo, tag: test_tag, file_path: test_file, download_folder: './')).to be_nil
+      donwloaded_file = described_class.download_file_from_tag(repository: test_repo, tag: test_tag, file_path: test_file, download_folder: './')
+      expect(donwloaded_file).to be_nil
```

Comment:
💄  Small typo 😉 
```suggestion
      downloaded_file = described_class.download_file_from_tag(repository: test_repo, tag: test_tag, file_path: test_file, download_folder: './')
      expect(downloaded_file).to be_nil
```

Reply:
Oh... more than 10 years of speaking/writing in English and I still can't write the word download right at first hahaha
Sorry for that 😅